wolfcrypt/src/ed448.c

```diff
@@ -179,37 +225,35 @@ static int ed448_sign_msg(const byte* in, word32 inLen, byte* out,
 
         /* step 1: create nonce to use where nonce is r in
            r = H(h_b, ... ,h_2b-1,M) */
-        ret = wc_Shake256Hash(key->k, ED448_KEY_SIZE, az, sizeof(az));
+        ret = ed448_hash(key, key->k, ED448_KEY_SIZE, az, sizeof az);
```

Comment:
For code consistency please use `sizeof(az)`.

Reply:
done